glossary.md

```diff
@@ -0,0 +1,22 @@
+# Glossary
+
+## Application
+
+A process tree launched inside a [container](#container).
+
+## Bundle
+
+A [directory structure](bundle.md) that is written ahead of time, distributed, and used to seed the runtime for creating [containers](#container) and launching [applications](#application).
```

Comment:
On Sat, Aug 29, 2015 at 02:26:41AM -0700, Lai Jiangshan wrote:
> But how to create other namespaces without starting any additional
> process?  Must be there some ways like network, but I can't find out
> any grace way currently.

They may not all have a way ;).  Any namespaces that turn out to
require a PID may require the runtime to store create-time
information, check that store when launching a new application, and
create any missing namespaces when launching that application.  That
gives us consistent client semantics (e.g. you can still join an
existing user namespace by listing it's path in application.json [1]).
And if/when the kernel gains the ability to create/destroy those
namespaces independently we can drop the associated runtime storage.

[1]: https://groups.google.com/a/opencontainers.org/forum/#!topic/dev/kPWu-ZavQjs